### ISSUE TITLE
Create a launch.json configuration for live preview

### ISSUE THREAD

**Issue by caiohamamura** [microsoft/vscode-livepreview#506]
Currently to run in debug mode, we have to open the `html` file and (provided you just installed it without any customization) you need to press open the command palette and run `Live Preview: Show Debug Preview (External Browser)`. 

Even though it is fairly easy to keybind it to `F5` and add a `When` expression to `editorLangId=='html'`, it is kind of a pretty default thing to run every other projects within VSCode in debug mode with the F5 keybinding. 

I am a teacher of `Web Development` for kids at High School of a vocational school, and it is pretty much their first steps in development and with VSCode. So it is kind of overwhelming (as we have Deep Freezed system) to always have to open the command palette and running `Live Preview: Show Debug Preview (External Browser)` or having to always set the keybinding just to see the output of a console.log within the `Debug Console`, when in other languages (Python, Dart, even C and C++) we just need to press F5 and we are already there.

**Comment by hawkerm** [comment]
Yeah, I have a build process which outputs an html file, so I'd like to setup the `launch.json` to run the compile command and then open/refresh the live preview window of that outputted file too.
